db/sqldb/suspect_db_test.go

```diff
@@ -65,7 +65,7 @@ var _ = Describe("Suspect ActualLRPs", func() {
 					Expect(err).NotTo(HaveOccurred())
 					_, _, err = sqlDB.ClaimActualLRP(ctx, logger, replacementGuid, replacementIndex, &replacementLRP.ActualLRPInstanceKey)
 					Expect(err).NotTo(HaveOccurred())
-					_, _, err = sqlDB.StartActualLRP(ctx, logger, &replacementLRP.ActualLRPKey, &replacementLRP.ActualLRPInstanceKey, &replacementLRP.ActualLRPNetInfo, model_helpers.NewActualLRPInternalRoutes(), nil, replacementLRP.GetRoutable())
+					_, _, err = sqlDB.StartActualLRP(ctx, logger, &replacementLRP.ActualLRPKey, &replacementLRP.ActualLRPInstanceKey, &replacementLRP.ActualLRPNetInfo, model_helpers.NewActualLRPInternalRoutes(), nil, replacementLRP.GetRoutable(), actualLRP.AvailabilityZone)
```

Comment:
Should the addition to the line read `replacedmentLRP.AvailabilityZone` instead? All of the other fields are using `replacementLRP`.

Alternatively, since we're not asserting on the value of the AZ anywhere in the test, would it make more sense to hard-code the value to `"some-zone"` or the empty string? This might be more consistent with example above where the AZ is not important for testing and used only to satisfy the interface for `StartActualLRP`.